extensions/curl/src/requests.tsx

```diff
@@ -0,0 +1,158 @@
+import { useEffect, useState } from "react";
+import {
+  Action,
+  ActionPanel,
+  Alert,
+  confirmAlert,
+  Icon,
+  List,
+  LocalStorage,
+  showToast,
+  Toast,
+  useNavigation,
+} from "@raycast/api";
+import axios from "axios";
+import ResultView from "./views/Result";
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const curlString = require("curl-string");
+
+interface Values {
+  key: string;
+  value: string;
+}
+
+interface RequestArg {
+  url: string;
+  payload: string;
+}
+
+export default function Requests() {
+  const [isLoading, setIsLoading] = useState<boolean>(false);
```

Comment:
It's better to set this to `true` initially to avoid flickering of the empty view.

Reply:
Made the changes in my fork but the PR is closed. How do you usually manage these changes afterward? Open a new PR? @thomaspaulmann

Reply:
Yeah just another PR